### ISSUE TITLE
Will this work on iOS? 

### ISSUE THREAD

**Issue by themakerman** [beast/react-native-scan-doc#1]
Does scanning and cropping of paper edges work on iOS?

**Comment by beast** [comment]
Yes, it will work. The reason I have not been able to work on the iOS is because it requires me to write opencv code for iOS platform， which I have not had the time to look into. But my Android code already provided the logic, you simply have to map it to iOS.

**Comment by ducpt2** [comment]
@themakerman Did you make it work? Thanks.

**Comment by EricPKerr** [comment]
What's the recommended way to get this scanning functionality on iOS and Android right now?

Thanks everyone

**Comment by ducpt2** [comment]
@EricPKerr Nope :(
